protocols/processDragenTrendanalysis.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash  
+
+set -e
+set -u
+
+ml jq/1.6
+
+#
+#still needs some work to make it fit in the pipeline...
+#
+
+readarray -t jsonFiles < <(find /groups/umcg-gst/tmp05/104832-034_A/Analysis/*/ -maxdepth 1 -mindepth 1 -type f -name "*.metrics.json")
+
+echo -e "Sample,Run,Date" > Dragen_runinfo.csv
+echo -e "Sample\tBatchName\ttotal_reads\tduplicate_reads\tunique_reads\tmean_insertsize\tfrac_min_20x_coverage\taverage_autosomal_coverage" > Dragen.csv
+
+for jsonFile in "${jsonFiles[@]}"
+do
+	seq_batch=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .sequencer.seq_batch ' | tr -d '"')
+	total_reads=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .mapping.total_reads')
+	duplicate_reads=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .mapping.duplicate_reads')
+	unique_reads=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .mapping.unique_reads')
+	mean_insertsize=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .insert.mean' )
+	frac_min_20x_coverage=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .coverage.roi1.frac_min_20x_coverage')
+	average_autosomal_coverage=$(less "${jsonFile}" | "${EBROOTJQ}/jq-linux64" '. | .coverage.roi1.average_autosomal_coverage')
```

Comment:
ik heb voor jq een nieuwe eb file gemaakt waarbij de jq-linux64 in de bin directory word gezet en het dus niet meer nodig is om EBROOTJQ ervoor te zetten. 
Dus je kunt gewoon jq-linux64 aanroepen

Reply:
ik zou persoonlijk `cat` gebruiken voor het lezen van de file ipv less

Reply:
bedankt, ik heb er cat van gemaakt en de ${EBROOTJQ} weggelaten. mocht alles op zn plek staan voor de dragen data ga ik wel alles "correct maken"